src/components/events-list/EventsList.tsx

```diff
@@ -0,0 +1,32 @@
+import React from 'react';
+import EventListing from '../event-listing/EventListing';
+
+// placeholder interface for now (will replace with real interface once hooked up to db)
+interface Event {
+  title: string;
+  date: string;
+  time: string;
+  description: string;
+  otherNotes?: string;
+  // TODO: use correct file type
+  thumbnail?: string;
+}
```

Comment:
Rather than defining this twice, let's create a types file. 

Create a new directory `ducks` with a file `src/containers/upcoming-events/ducks/types.ts`. In types, define this interface and export it, then use it anywhere it's required elsewhere.